### PR TITLE
Added workspace aliases possibility

### PR DESCRIPTION
**Pull request by scholz-timo**

### CODE REVIEW

File: steps/helm/src/types/definition.ts [hunk 1]
```diff
@@ -8,6 +8,10 @@ export const definitions = {
     is_service: {
       type: 'boolean'
     },
+    // TODO: Alias should only be avilable for is_service = true.
+    alias: {
```
Comment (scholz-timo) [inline steps/helm/src/types/definition.ts:12]: @FCQCarryOut kannst du das schema hier anpassen?

**Comment by scholz-timo** [comment]
I will create a pre release from this branch to test it in the other project.

File: steps/helm/src/types/definition.ts (hunk 1 above)
Comment (FCQCarryOut) [reply steps/helm/src/types/definition.ts:12]: done in b601318dc8b80215bc95eb8cc7fde3935c2918ad

Comment (scholz-timo) [reply steps/helm/src/types/definition.ts:11]: Noice, i removed todo.

File: steps/helm/src/HelmGeneratorProvider.ts [hunk 2]
```diff
@@ -226,15 +240,20 @@ export class HelmGeneratorProvider implements IStep {
...
+    return this.workspaceAliasGenerator(workspace, root);
+  }
+
+  private async getContext(workspace: IWorkspace, map: HelmFileResult, root: string): Promise<Record<'global' | 'scoped', ExtendedSchemaType> | undefined> {
     const [helmFiles, workspaceMapping] = map;
 
-    const childScopes = (await Promise.all((workspaceMapping.get(workspace) ?? []).map((workspace) => this.getContext(workspace, map)))).filter(<T>(value: T | undefined): value is T => value !== undefined);
+    const childScopes = (await Promise.all((workspaceMapping.get(workspace) ?? []).map((workspace) => this.getContext(workspace, map, root)))).filter(<T>(value: T | undefined): value is T => value !== undefined);
 
     // TODO: Combine bases.
```
Comment (scholz-timo) [inline steps/helm/src/HelmGeneratorProvider.ts:258]: Remove todo.

**Review by FCQCarryOut — APPROVED** [review]

File: steps/helm/src/HelmGeneratorProvider.ts (hunk 2 above)
Comment (scholz-timo) [reply steps/helm/src/HelmGeneratorProvider.ts]: Done

**Pull request merged**